### PR TITLE
feat: update project structure to only include core components

### PR DESCRIPTION
**Pull request by pinussilvestrus**
* Introduces `@bpmn-io/properties-panel` as core properties panel library.
* Moves all BPMN related stuff to https://github.com/bpmn-io/bpmn-properties-panel/pull/1

Related to #5 

Steps that need to be done after
* <del>rename project to `bpmn-io/properties-panel` (or something different if it doesn't fit)</del> (done)
* <del>update repo URLs</del> (done)
* <del>move BPMN specific [issues](https://github.com/bpmn-io/properties-panel-next/issues) to `bpmn-io/bpmn-properties-panel`</del> (done)
* release a first version

### CODE REVIEW

**Comment by pinussilvestrus** [comment]
We added the `disabled` support for selects and text fields inside this PR, since they already follow the new project structure (cf.  #78, #79)

File: test/spec/components/Select.spec.js [hunk 1]
```diff
@@ -75,6 +75,60 @@ describe('<Select>', function() {
   });
 
 
+  describe('options', function() {
+
+    it('should render options non-disabled per default', function() {
```
Comment (philippfromme) [inline test/spec/components/Select.spec.js:80]: Maybe change to _should render options enabled_ (cf. https://github.com/bpmn-io/properties-panel/pull/77/commits/3a66f3c7187c9257f578bdba4268c08ecb009844#diff-8a7dd5b0e7a682b72766dfbe7dc3696aa7ad62696c049e971c0602cf452f62eeR221).

File: test/spec/components/Select.spec.js [hunk 2]
```diff
@@ -75,6 +75,60 @@ describe('<Select>', function() {
...
+      const select = domQuery('.bio-properties-panel-select', result.container);
+
+      // then
+      const optionA = domQuery('option[value="A"]', select);
+
+      expect(optionA.disabled).to.be.false;
+    });
+
+
+    it('should render option not-disabled if set explicitly', function() {
```
Comment (philippfromme) [inline test/spec/components/Select.spec.js:97]: Maybe change to _should render option enabled_ (cf. https://github.com/bpmn-io/properties-panel/pull/77/commits/3a66f3c7187c9257f578bdba4268c08ecb009844#diff-8a7dd5b0e7a682b72766dfbe7dc3696aa7ad62696c049e971c0602cf452f62eeR221).

**Review by philippfromme — COMMENTED** [review]
One small remark.

File: test/spec/components/Select.spec.js (hunk 1 above)
Comment (pinussilvestrus) [reply test/spec/components/Select.spec.js]: Updated it to have a clearer wording (_enabled_ in favor of _non-disabled_)

Comment (pinussilvestrus) [reply test/spec/components/Select.spec.js]: Thanks!

**Comment by pinussilvestrus** [comment]
Updated ✌️

**Comment by philippfromme** [comment]
Approved. ✅

**Pull request merged**